recipes/instaloader/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "instaloader" %}
+{% set version = "4.9.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/instaloader-{{ version }}.tar.gz
+  sha256: 16040c170fb5230c1981a47e1990261e3c0ecffe0417be95fa265632244e7c01
+
+build:
+  entry_points:
+    - instaloader=instaloader.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - requests >=2.4
+
+test:
+  imports:
+    - instaloader
+  commands:
```

Comment:
Please add a check for the command line app like `- instaloader -h` under `commands`

Reply:
Added it in cc98de4